### PR TITLE
BB Fuel Endpoint Fixes

### PR DESCRIPTION
**Pull request by vinibackbase**
- Fixing Users;
- Fixing Payments;

Screen Shot 2020-03-13 at 12 23 42

### CODE REVIEW

File: src/main/java/com/backbase/ct/bbfuel/client/payment/PaymentOrderPresentationRestClient.java [hunk 1]
```diff
@@ -16,14 +16,15 @@
     private final BbFuelConfiguration config;
 
     private static final String SERVICE_VERSION = "v2";
+    private static final String SERVICE = "client-api";
```
Comment (polarfish) [inline src/main/java/com/backbase/ct/bbfuel/client/payment/PaymentOrderPresentationRestClient.java]: I suggest to call this constant `CLIENT_API`

File: src/main/java/com/backbase/ct/bbfuel/client/user/UserIntegrationRestClient.java [hunk 2]
```diff
@@ -23,11 +23,13 @@
 
     private final BbFuelConfiguration config;
     private static final String SERVICE_VERSION = "v2";
+    private static final String SERVICE = "client-api";
```
Comment (polarfish) [inline src/main/java/com/backbase/ct/bbfuel/client/user/UserIntegrationRestClient.java]: Why do we use "client-api" in User**Integration**RestClient?

File: src/main/java/com/backbase/ct/bbfuel/client/user/UserPresentationRestClient.java [hunk 3]
```diff
@@ -17,6 +17,7 @@
     private final BbFuelConfiguration config;
 
     private static final String SERVICE_VERSION = "v2";
+    private static final String SERVICE = "client-api";
```
Comment (polarfish) [inline src/main/java/com/backbase/ct/bbfuel/client/user/UserPresentationRestClient.java]: I suggest to call this constant `CLIENT_API`

File: src/main/java/com/backbase/ct/bbfuel/client/user/UserIntegrationRestClient.java (hunk 2 above)
Comment (vinibackbase) [reply src/main/java/com/backbase/ct/bbfuel/client/user/UserIntegrationRestClient.java]: Without the addition of "client-api" the services are returning 404.

File: src/main/java/com/backbase/ct/bbfuel/client/payment/PaymentOrderPresentationRestClient.java (hunk 1 above)
Comment (vinibackbase) [reply src/main/java/com/backbase/ct/bbfuel/client/payment/PaymentOrderPresentationRestClient.java]: done

File: src/main/java/com/backbase/ct/bbfuel/client/user/UserPresentationRestClient.java (hunk 3 above)
Comment (vinibackbase) [reply src/main/java/com/backbase/ct/bbfuel/client/user/UserPresentationRestClient.java]: done

File: src/main/java/com/backbase/ct/bbfuel/client/user/UserIntegrationRestClient.java (hunk 2 above)
Comment (polarfish) [reply src/main/java/com/backbase/ct/bbfuel/client/user/UserIntegrationRestClient.java]: We have `UserPresentationRestClient.java` for calling `client-api` and `UserIntegrationRestClient` for calling `integration-api`.
I do not understand why do we mix responsibilities.

Comment (vinibackbase) [reply src/main/java/com/backbase/ct/bbfuel/client/user/UserIntegrationRestClient.java]: Indeed we do, but it doesn't work. Please try on this endpoint if you want to be sure:
http://user.2383.dbs.backbase.test/integration-api/v2/users

Comment (polarfish) [reply src/main/java/com/backbase/ct/bbfuel/client/user/UserIntegrationRestClient.java]: So our integration API is exposed via `/client-api` as well (I just thought we are on the lean architecture already). 
Please rename the constant to be `CLIENT_API`(like others).

**Pull request merged**